tests/ai_hub/utils.py

```
@@ -1068,3 +1069,23 @@ def get_latest_job_pod(admin_client: DynamicClient, job: Job) -> Pod:
+) -> bool:
+    """Determine if a name should be included based on include/exclude glob patterns."""
+    matches_included = any(fnmatch(name, pattern) for pattern in included_patterns) if included_patterns else True
+    matches_excluded = any(fnmatch(name, pattern) for pattern in excluded_patterns) if excluded_patterns else False
+    return matches_included and not matches_excluded
+
+
+def execute_authenticated_post(url: str, token: str, files: dict[str, tuple[str, str, str]]) -> dict[str, Any]:
+    """Execute an authenticated POST with multipart/form-data and return parsed JSON."""
+    headers = {"Authorization": f"Bearer {token}"}
+    LOGGER.info(f"Executing POST: {url}")
+    response = requests.post(url=url, headers=headers, files=files, verify=False, timeout=60)
```

Ruff (0.15.18)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)